server/utils/rating.util.js

```diff
@@ -0,0 +1,39 @@
+import db from '../models';
+
+const { Article } = db;
+
+class Rating {
+  /**
+   * Check the existence of an article
+   * @param {string} id
+   * @returns {boolean} aricleExists
+   */
+  static async doesArticleExist(id) {
+    return true;
+  }
+
+  /**
+   * Get the claps for an article
+   * @param {string} id
+   * @returns {number} claps
+   */
+  static async getArticleClaps(id) {
+    return 30;
+  }
+
+  /**
+   * Get the claps for an article
+   * @param {number} claps
+   * @returns {number} rating
+   */
+  static getArticleRating(claps) {
```

Comment:
Seems we are good to go with this algorithm.

Reply:
Hi @waleCloud We have implemented this algorithm with switch case